src/sentry/integrations/gitlab/integration.py

```diff
@@ -273,6 +273,7 @@ class GitlabIntegrationProvider(IntegrationProvider):
             IntegrationFeatures.ISSUE_BASIC,
             IntegrationFeatures.COMMITS,
             IntegrationFeatures.STACKTRACE_LINK,
+            IntegrationFeatures.CODEOWNERS,
```

Comment:
Adding this here now is going to make it show up for everyone, and because it seems like since the feature flag isn't constructed with the `integrations-` prefix, it's going to show up as a free feature for everyone. Might want to confirm that by testing it out on getsentry though.